spinn_front_end_common/interface/interface_functions/application_runner.py

```diff
@@ -83,7 +61,8 @@ def run_application(
         buffer_manager.load_initial_buffers()
 
         # wait till external app is ready for us to start if required
-        notifier.wait_for_confirmation()
+        if wait_on_confirmation:
```

Comment:
This test should be moved into real `NotificationProtocol`, as that already has the `wait_on_confirmation` flag (as `self._wait_for_read_confirmation`).

Reply:
Yes.  So nuked the repeat param in Application Runner/

Added the test to wait_for_confirmation but also to send_read_notification to not save a wait that will never be used.